.github/workflows/build_test.yml

```diff
@@ -0,0 +1,116 @@
+name: R
+
+on:
+  pull_request:
+    branches: [ "develop" ]
+
+jobs:
+  Building-testing-package:
+    runs-on: ubuntu-latest
+    # if: github.event.pull_request.merged == true || github.event.review.state == 'approved'
+    steps:
+      - uses: actions/checkout@v2
+
+      ### RELEASE VERSION SECTION ###
+      - name: Get current version (MAJOR.MINOR) from DESCRIPTION file
+        id: get_version
+        run: |
+          # Extract version from DESCRIPTION file
+          VERSION=$(grep '^Version:' DESCRIPTION | awk '{print $2}')
+          MAJOR=$(echo $VERSION | cut -d '.' -f 1)
+          MINOR=$(echo $VERSION | cut -d '.' -f 2)
+          
+          # Output MAJOR and MINOR to GitHub environment
+          echo "MAJOR=$MAJOR" >> $GITHUB_ENV
+          echo "MINOR=$MINOR" >> $GITHUB_ENV    
+          echo "The version from DESCRIPTION: $MAJOR.$MINOR"
+
+      - name: Get latest PATCH version from GitHub releases
+        id: get_latest_patch
+        run: |
+          # Get the list of existing releases
+          releases=$(curl -s "https://api.github.com/repos/${{ github.repository }}/releases")
+      
+          # Find the latest release with the same MAJOR.MINOR version
+          latest_patch=$(echo "$releases" | jq -r --arg MAJOR "$MAJOR" --arg MINOR "$MINOR" '
+            [
+              .[] | select(.tag_name | startswith("\($MAJOR).\($MINOR).")) | .tag_name
+            ] | map(split(".")[2] | tonumber) | max // 0
+          ')
+      
+          # Increment the PATCH version
+          PATCH=$((latest_patch + 1))
+      
+          # Output PATCH to the GitHub environment
+          echo "PATCH=$PATCH" >> $GITHUB_ENV    
+          echo "New Patch number: $PATCH"
+
+      # Combine the version components
+      - name: Set and display new release version
+        run: |
+          VERSION="$MAJOR.$MINOR.$PATCH"
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "New version is ${{ env.VERSION }}"
+      ### END OF RELEASE VERSION SECTION ###
+      
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: '4.1.1'
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libcurl4-openssl-dev        
+          sudo apt-get install -y libharfbuzz-dev libfribidi-dev libfreetype6-dev
+          sudo apt-get install -y libudunits2-dev libgdal-dev libgeos-dev libproj-dev
+          sudo apt-get install -y pkg-config
+
+      - name: Install R dependencies
+        run: |
+          install.packages("remotes")
+          remotes::install_deps(dependencies = TRUE)
+        shell: Rscript {0}
+
+      - name: Build package
```

Comment:
I think eventually we would also want to rebuild the vignettes and the manaul.

Reply:
Sorry I don't understand about vignette. Manual, I agree.

Reply:
@maasva you can add it to our backlog, but not for this PR

Reply:
Alright. Vignette is like an example of things for AFAIK. Why not for this PR though? Isn't it as easy as just removing the -- tags?

Reply:
Because I don't know anything about it, like removing -- where?

Reply:
It was also not planned before (no description / acceptance criteria)
and I want to focus on something else, like 2 config files